### PR TITLE
Create a new Python fuzzing project for tensorflow

### PR DESCRIPTION
**Pull request by av8ramit**
Create a new Python fuzzer under `projects/tensorflow-py`. 

Based on the following [integration directions](https://google.github.io/oss-fuzz/getting-started/new-project-guide/python-lang/).

@mihaimaruseac

### CODE REVIEW

File: projects/tensorflow-py/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+FROM gcr.io/oss-fuzz-base/base-builder
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        curl \
+        rsync \
+        vim \
+        && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+RUN python3 -m pip install numpy
```
Comment (mihaimaruseac) [inline projects/tensorflow-py/Dockerfile:26]: Nit: We can remove this, will get installed by TF pip

**Review by mihaimaruseac — APPROVED** [review]

**Review by mihaimaruseac — APPROVED** [review]

File: projects/tensorflow-py/build.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+#
+################################################################################
+
+# Build and install project (using current CFLAGS, CXXFLAGS). This is required
+# for projects with C extensions so that they're built with the proper flags.
+python3 -m pip install tensorflow
+
+# Rename to avoid the following: https://github.com/tensorflow/tensorflow/issues/40182
+mv $SRC/tensorflow/tensorflow $SRC/tensorflow/tensorflow_src
+# Build fuzzers into $OUT. These could be detected in other ways.
```
Comment (inferno-chromium) [inline projects/tensorflow-py/build.sh:24]: new line before this line.

File: projects/tensorflow-py/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 Google Inc.
```
Comment (inferno-chromium) [inline projects/tensorflow-py/Dockerfile:1]: 2021 in all files.

File: projects/tensorflow-py/build.sh [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+for fuzzer in $(find $SRC -name '*_fuzz.py'); do
+  fuzzer_basename=$(basename -s .py $fuzzer)
+  fuzzer_package=${fuzzer_basename}.pkg
+  # To avoid issues with Python version conflicts, or changes in environment
```
Comment (inferno-chromium) [inline projects/tensorflow-py/build.sh:28]: Remove all these comments, these were more explanations in docs, can remove this and one below.

**Review by inferno-chromium — CHANGES_REQUESTED** [review]
LGTM with minor nits to fix.

**Review by inferno-chromium — APPROVED** [review]

**Pull request merged**